assets/src/stories-editor/components/editor-carousel/index.js

```diff
@@ -57,9 +60,14 @@ const EditorCarousel = () => {
 			wrapper.current.style.display = 'none';
 		} else {
 			wrapper.current.style.display = '';
-			wrapper.current.style.transform = `translateX(calc(50% - ${ PAGE_BORDER }px - ${ ( STORY_PAGE_INNER_WIDTH + STORY_PAGE_MARGIN ) / 2 }px - ${ ( currentIndex ) * STORY_PAGE_MARGIN }px - ${ currentIndex * STORY_PAGE_INNER_WIDTH }px))`;
+
+			if ( isRTL ) {
+				wrapper.current.style.transform = `translateX(calc(-50% - ${ PAGE_BORDER }px + ${ ( STORY_PAGE_INNER_WIDTH + STORY_PAGE_MARGIN ) / 2 }px + ${ ( currentIndex ) * STORY_PAGE_MARGIN }px + ${ currentIndex * STORY_PAGE_INNER_WIDTH }px))`;
```

Comment:
Is there any reason why we're off-loading this calculation to css? Why not calculate the pixels in JS and only `calc(50% + ${x}px)`? Then we can also un-duplicate the logic, by setting `const factor = isRTL ? -1 : 1;` somewhere.

Reply:
Sure that might also work, I'll give it a try. Might make it more readable 👍

Reply:
Actually, not so sure 🤔

Reply:
I tried a simplification in 36d5a4a. Feel free to revert if you don't find it simpler 🤷‍♂.